docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1575,3 +1575,108 @@ module.exports = () => ({
   },
 });
 ```
+
+### Role-Based Access Control <BronzeBadge link="https://strapi.io/pricing"/> <SilverBadge link="https://strapi.io/pricing"/> <GoldBadge link="https://strapi.io/pricing" withLinkIcon/>
+
+::: warning 🚧 This API is considered unstable for now.
+<br>
+:::
+
+Role-Based Access Control (RBAC) is an approach to restricting access to some users. In a Strapi application, users of the admin panel are administrators. Their roles and permissions are [configured in the admin panel](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md). In addition to the 3 default roles (Author, Editor, and Super Admin), it's possible to create custom conditions to match specific needs of roles and permissions management. This requires an Enterprise Edition with at minimum a Bronze licence plan.
```

Comment:
The custom conditions are not really in addition of the 3 defaults roles, they are in addition of the roles in general. Is there another way to say this ?

Reply:
Thanks @petersg83 
I tried to simply [reframe it like this](https://github.com/strapi/documentation/pull/187/commits/6381418b7939486c4705f2674826d5d413b8b9f8), to remove the causal link between the two sentences. Is it better now?